### DISCUSSION TITLE
UserBots, develop a limited official SDK?

### DISCUSSION THREAD

**Discussion by andrei0x309** [discordjs/discord.js#7186] [Q&A] [answered]
I noticed that Discord adjusted the attitude towards usersBots which I think in most cases is a good thing to prevent spam and other problems. (although I don't know how much it helped since I receive much spam every day) 

But don't you think that Discord should provide a limited model for user automatization? Because there are many legit cases, for example, in my case, in the past I could have used UserBots for the following scenario:

Someone would give me a DM, and a server would check if my main PC is online and in case it wasn't online it would reply that I am currently offline.

I guess is a delicate act to balance dev experience and at the same time prohibit harmful behavior but I think some limited degree of user programmatic interaction would be a net benefit for Discord since a lot of users complain using the official Discord forum about missing features and this can alleviate some missing features.

**Comment by iCrawl** [comment, marked as answer]
No, and I don't see how it concerns us as a 3rd party library.
